README.md

```diff
@@ -61,7 +61,7 @@ A possible solution to this would be to create an extra APIs implementing the mi
 [Composer](http://getcomposer.org/download/) users can simply run:
 
 ```bash
-$ php composer.phar require kbsali/redmine-api:~1.0
+$ php composer.phar require kbsali/redmine-api
```

Comment:
```suggestion
$ composer require kbsali/redmine-api
```
? the .phar is the very old way of playing with composer, don't you think?

Reply:
I'm still using the `composer.phar` in almost all projects to not have a globally dependency. :blush:
I would instead look at the possible users of the library. For advanced user composer will be clear. But in the README.php and example.php it suggests that this library aims also beginners.